### PR TITLE
model users and migrations

### PR DESCRIPTION
**Pull request by DiegoWolox**
## Summary

The user model and the respective migration are created

## Trello Card

[Modelo de usuario](https://trello.com/c/KSzgwnhz/21-modelo-de-usuario)

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
```
Comment (marioportela1993) [inline app/models/user.js:2]: Set up at the model level the primary key, require fields and these type of configurations.

File: migrations/migrations/20210211132634-create-user.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
```
Comment (marioportela1993) [inline migrations/migrations/20210211132634-create-user.js:6]: Excellent, set up this configs in the model also

File: migrations/migrations/20210211132634-create-user.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      },
+      first_name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      last_name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      email: {
```
Comment (marioportela1993) [inline migrations/migrations/20210211132634-create-user.js:19]: Put this field as required

File: app/models/user.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    'user',
+    {
+      firstName: DataTypes.STRING,
+      lastName: DataTypes.STRING,
+      email: DataTypes.STRING,
+      password: DataTypes.STRING
+    },
+    {
+      underscored: true
```
Comment (marioportela1993) [inline app/models/user.js]: We usually set the `tableName: users`,  due to in some cases the table name could be different to the model name

Comment (marioportela1993) [inline app/models/user.js]: Set also `timestamps: true`

**Review by marioportela1993 — CHANGES_REQUESTED** [review]
Excellent work! check the comments and apply them to approve the PR

File: app/models/user.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'user',
```
Comment (emeneses4) [inline app/models/user.js]: The model name must be with the first letter in uppercase

**Review by marioportela1993 — APPROVED** [review]
Nice! For me is ok to merge

**Pull request merged**